nand2tetris.el

```diff
@@ -161,13 +191,13 @@ See URL `http://www.nand2tetris.org/software.php'."
    (nand2tetris//get-compare-file (current-buffer))))
 
 (defun nand2tetris/tests-current-hdl ()
-  "Run `HardwareSimulator.sh' on current tst file."
+  "Run `HardwareSimulator.bat' on current tst file."
   (interactive)
   (save-buffer)
   (shell-command (concat (nand2tetris-hardware-simulator) " " (nand2tetris//get-current-test-file))))
 
 (defun nand2tetris/tests-current-hdl-elsewhere ()
-  "Run `HardwareSimulator.sh' on current tst file, but on another locaion.
+  "Run `HardwareSimulator.bat' on current tst file, but in another location.
```

Comment:
instead of the extension name. Can we use this wording instead? `Run HardwareSimulator script` that way it's agnostic